### PR TITLE
Add convenience methods to DataTable.

### PR DESCRIPTION
**Pull request by klshrinidhi**
Addresses #797 and is based on #801.

### CODE REVIEW

File: OpenSim/Common/DataTable.h [hunk 1]
```diff
@@ -311,6 +344,11 @@ class DataTable_ : public AbstractDataTable {
         return _depData.col((int)index);
     }
 
+    /** Get dependent Column which has the given column label.                */
+    VectorView getDependentColumn(const std::string& columnLabel) {
+        return _depData.col(getColumnIndex(columnLabel));
+    }
+
```
Comment (aseth1) [inline OpenSim/Common/DataTable.h]: Nice. Is there similar convenience access to the independent column data? For TimeSeriesTable you could also add `getTimes()` or `getTimeColumn()` which returns the independent column.

**Comment by chrisdembia** [comment]
Looks good to me.

File: OpenSim/Common/DataTable.h (hunk 1 above)
Comment (chrisdembia) [inline OpenSim/Common/DataTable.h]: Could be a separate PR for a `getTimes()` or similar.
